models/intermediate/int_recharge__customer_daily_rollup.sql

```diff
@@ -3,7 +3,7 @@ with calendar as (
     from {{ ref('int_recharge__calendar_spine') }}
 
 ), customers as (
-    select customer_id, created_at
+    select customer_id, created_at 
```

Comment:
Small suggestion to be in line with style standards.
```suggestion
    select 
         customer_id, 
         created_at 
```